sherlock/resources/data.json

```diff
@@ -1203,6 +1209,12 @@
     "urlMain": "https://letterboxd.com/",
     "username_claimed": "blue"
   },
+  "LibraryThing": {
+    "errorType": "status_code",
```

Comment:
Looks like checking for the status code will result in false positives. Please update the error type to "message" and set the errorMsg to "This user doesn't exist"

![firefox_b1dTAYorYM](https://github.com/sherlock-project/sherlock/assets/27065646/9e2f33a1-9079-4b02-b5ce-78f61bbc35cb)

Reply:
The requested changes have been made and updated.